Rules/UseConsistentIndentation.cs

```diff
@@ -195,15 +195,20 @@ public override IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string file
                             {
                                 ++tempIndentationLevel;
                             }
-                            else
+
+                            // check for comments in between multi-line commands with line continuation
+                            if (k > 2 && tokens[k - 1].Kind == TokenKind.NewLine
+                                && tokens[k - 2].Kind == TokenKind.Comment)
                             {
-                                // Ignore comments
-                                // Since the previous token is a newline token we start
-                                // looking for comments at the token before the newline token.
                                 int j = k - 2;
```

Comment:
`j` here could also do with a name change perhaps

Reply:
I'd imagine something like `tokenIndex` for `k` and `lastLineContinuationIndex` for `j`

Reply:
I went for `searchForPrecedingLineContinuationIndex` for `j` as the tokens value gets changed quite a bit until it maybe reaches a line continuation token.